content/riak/kv/2.1.4/using/cluster-operations/update-search-schema-reindex-data.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Updating & Reindexing Search Schemas"
+description: ""
+project: "riak_kv"
+project_version: "2.1.4"
+menu:
+  riak_kv-2.1.4:
+    name: "Updating / Reindexing Search Schemas"
+    identifier: "cluster_operations_search_schemas"
+    weight: 117
+    parent: "managing_cluster_operations"
+toc: true
+canonical_link: "https://docs.basho.com/riak/kv/latest/using/cluster-operations/update-reindex-search-schema"
+---
+
+The following page provides instruction on updating [search schemas](../../../developing/usage/search-schemas) as well as reindexing existing data.
+
+## Update the Search Schema
+
+First, make updates to the schema file, which is typically found here `$RIAK_HOST/search/index/famous`.
```

Comment:
1. "make updates to" should be "update the schema file"
2. What kind of updates? Why?